### PR TITLE
Changed axes definitions to not allow user changes to offset/Dir via the BeckhoffAxisNoOffset class

### PR DESCRIPTION
**Pull request by Mbosum**
## Description

 Prevent writes to DIR/OFF from the UI and Python from users.
## Motivation and Context

Make UI for upcoming TMO MRCO experiments as simple as possible, including removing possibilities for users to change settings that generate confusion or errors in motion and position RBVs.

## How Has This Been Tested?

I tested the module with flake8 and pytest and both came back clean.

(pcds-4.1.4) psdev04:pcdsdevices bosum123$ flake8 mrco_motion.py
(pcds-4.1.4) psdev04:pcdsdevices bosum123$

(pcds-4.1.4) psdev04:pcdsdevices bosum123$ pytest mrco_motion.py
================================================= test session starts ==================================================
platform linux -- Python 3.8.8, pytest-6.2.3, py-1.10.0, pluggy-0.13.1
PyQt5 5.12.3 -- Qt runtime 5.12.9 -- Qt compiled 5.12.9
benchmark: 3.4.1 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /reg/g/pcds/epics-dev/bosum123/hutch_python/pcdsdevices
plugins: timeout-1.4.2, qt-3.3.0, benchmark-3.4.1
collected 0 items

=================================================== warnings summary ===================================================
../../../../../pyps/conda/py36/envs/pcds-4.1.4/lib/python3.8/site-packages/past/builtins/misc.py:45
/reg/g/pcds/pyps/conda/py36/envs/pcds-4.1.4/lib/python3.8/site-packages/past/builtins/misc.py:45: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
from imp import reload

-- Docs: https://docs.pytest.org/en/stable/warnings.html
================================================== 1 warning in 4.23s ==================================================
(pcds-4.1.4) psdev04:pcdsdevices bosum123$

image
## Where Has This Been Documented?

1. Pre-release notes.
2. Docstring update.

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [ ] Test suite passes on travis
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

**Review by ZLLentz — APPROVED** [review]
LGTM

**Review by klauer — APPROVED** [review]

**Comment by ZLLentz** [comment]
The automated travis build had one minor complaint:
https://travis-ci.com/github/pcdshub/pcdsdevices/jobs/511188894#L205
```
pcdsdevices/mrco_motion.py:19:80: E501 line too long (80 > 79 characters)
```

**Comment by Mbosum** [comment]
Done.

**Comment by ZLLentz** [comment]
I will merge after the travis finishes/passes

File: pcdsdevices/mrco_motion.py [hunk 1]
```diff
@@ -16,7 +16,8 @@ class MRCO(BaseInterface, Device):
     MRCO Motion Class
 
     This class controls motors fixed to the MRCO Motion system for the IP1
-    endstation in TMO.
+    endstation in TMO. Offset/Dir changes on Python/UI level have been
```
Comment (ZLLentz) [inline pcdsdevices/mrco_motion.py:19]: https://travis-ci.com/github/pcdshub/pcdsdevices/jobs/511197151#L205
```suggestion
    endstation in TMO. Offset/Dir changes on Python/UI level have been
```

Comment (Mbosum) [reply pcdsdevices/mrco_motion.py]: ok should be fixed now.

**Pull request merged**